edm4eic.yaml

```diff
@@ -419,7 +421,6 @@ datatypes:
     OneToOneRelations:
       - edm4eic::Trajectory                     trajectory      // Trajectory of this track
     OneToManyRelations:
-      - edm4eic::Measurement2D measurements      // Measurements that were used for this track
       - edm4eic::Track      tracks            // Tracks (segments) that have been combined to create this track
```

Comment:
This is the problematic part here. How can we get people to start using the new Track structures when they get implemented if you will have removed the functionality.

Suggestion:
- put `edm4eic::Track::measurements` back,
- rename `edm4eic::Trajectory::{measurements,outliers}` to append `_deprecated` and indicate in the comment what should be used.

Reply:
I will do what you suggested. 

But I think we need a bit more discussion on this. In the previous trajectory structure (which I try to revive), we store good measurements (which used in track fit) and outliers (not included in track fit due to chi2 cut etc but is the best candidate on that layer).  In the current analysis we can learn from the outlier position and chi2 on how to improve tracking.  
In this new track structure, we only have one member "measurements". If that means we won't store outliers, then we miss this piece of useful info. If we want to store both track measurements and outliers into it, then we need to add an array of state flag to distinguish them.